workflows/diagnostics/tests/prognostic/_regtest_outputs/test_load_diag_data.test_Simulations[simulation0].out

```diff
@@ -1,51 +1,44 @@
 xarray.Dataset {
 dimensions:
-	time = 8 ;
```

Comment:
Basing the regression test off of output from a previous e2e run changed the output values.  Updating the regression file to match current.

Reply:
Yeah, at some point xarray changed the order it printed these dims in... maybe our previous regtest data used output generated with earlier version of xarray.